From 03dc7fc357cc9ffa853a1f64eb34fb550be1f4a5 Mon Sep 17 00:00:00 2001
From: Maryna Malakhova <maryna.malakhova@globallogic.com>
Date: Wed, 7 Apr 2021 13:36:00 +0300
Subject: [PATCH] 09_DT_RTC: Add source of rtc0 adding overlay

Add source of rtc0 adding overlay to the device tree
Build command: dtc -@ -O dtb -o BBB_black_rtc0.dtbo BBB_black_rtc0.dts
BBB_black_rtc0.dtbo compiles without any errors
I have some questions about it`s bootloading by netboot
I would be appreciate for a help

Signed-off-by: Maryna Malakhova <maryna.malakhova@globallogic.com>
---
 arch/arm/boot/dts/BBB_black_rtc0.dts | 20 ++++++++++++++++++++
 1 file changed, 20 insertions(+)
 create mode 100644 arch/arm/boot/dts/BBB_black_rtc0.dts

diff --git a/arch/arm/boot/dts/BBB_black_rtc0.dts b/arch/arm/boot/dts/BBB_black_rtc0.dts
new file mode 100644
index 000000000000..1a66a89a3b45
--- /dev/null
+++ b/arch/arm/boot/dts/BBB_black_rtc0.dts
@@ -0,0 +1,20 @@
+/dts-v1/;
+/plugin/;
+
+/ {
+fragment@0 {
+	target = <&GLCamp>;
+		__overlay__ {
+		i2c2 {
+		#address-cells = <1>;
+                #size-cells = <0>;
+		compatible = "ti,beaglebone", "ti,beaglebone-black";
+		ds3231: rtc@0 {
+			compatible = "maxim, ds3231";
+			reg = <0x68>;
+				};
+			};
+		};
+	};
+};
+
--
2.25.1

